RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+Improves the following for types converted from pydantic BaseModel.
+Adds mypy extension support for to_pydantic and from_pydantic methods.
+Adds typing stubs for IDE support.
```

Comment:
```suggestion
Release type: minor

Improves the following for types converted from pydantic BaseModel.
Adds mypy extension support for `to_pydantic` and `from_pydantic` methods.
Adds typing stubs for IDE support.
```

Reply:
> Adds typing stubs for IDE support.

Not sure what this means to be honest 🤔  I dont' see any type stubs :)

Reply:
i was refering to `StrawberryTypeFromPydantic` the protocol class. XD

Reply:
can we rephrase it? :D or remove it if not needed 😊